docs/source/sysadmin_guide/configuring_permissions.rst

```diff
@@ -208,13 +253,28 @@ rules and is evaluated in order. If the public key is denied, the connection
 will be rejected. For more information, please look at the
 :doc:`Identity Transaction Family <../transaction_family_specifications/identity_transaction_family>`.
 
-Like above, run the following command to set the role for network to permit all:
+1. As in the previous procedure (see :ref:`config-onchain-txn-perm-label`),
+   make sure that the Identity transaction processor and the REST API are
+   running, and that your public key is in the
+   ``create sawtooth.identity.allowed_keys`` setting.
 
-.. code-block:: console
+   .. code-block:: console
+
+     $ sawset proposal create sawtooth.identity.allowed_keys=02b2be336a6ada8f96881cd55fd848c10386d99d0a05e1778d2fc1c60c2783c2f4
 
-    $ sawtooth identity role create network policy_1
+#. If necessary, create a new policy.
 
-To see the role in state, run the following command:
+   This procedure assumes that ``policy_1`` already exists on-chain, as created
```

Comment:
Can you add a comment about how you could use the same policy but in practice these would be different keys and it does not really make sense to do so? For example, you would not put transaction signer keys into the network policies.

Reply:
Instead of an additional note, I think we should change this procedure to not reuse `policy_1`.  Otherwise, it might imply that the same policy is being being reused on purpose.